modules/accounts/accounts.js

```diff
@@ -18,7 +18,13 @@ class Accounts{
 
     async setProvider(args){
         Logger.verbose(`Command : tezster set-rpc-node ${args}`);
-        this.setProviderAccounts(args.newNodeProvider);
+        let providerToSet;
+        if(args.newCustomNodeProvider) {
```

Comment:
this can be simplified to 
let providerToSet = args.newCustomNodeProvider ? args.newCustomNodeProvider : args.newNodeProvider